internal/stores/hostdb.go

```diff
@@ -660,20 +660,50 @@ func (ss *SQLStore) ProcessConsensusChange(cc modules.ConsensusChange) {
 				announcement: ha,
 			})
 		})
+		// Update RevisionHeight and RevisionNumber for our contracts.
```

Comment:
`nit` do we care about  reverted blocks?

Reply:
This should work for reorgs assuming that the revision or proof get resubmitted. Because we just override the fields again. I think that's good enough for now.